tests/test_transformers.py

```diff
@@ -49,6 +50,11 @@ def create_gridsearch(self, transformer) -> GridSearchCV:
         )
 
 
+@parametrize_with_checks([Binner(), Select()])
```

Comment:
This should probably go in a new branch - not related to updating docstrings :wink: